packages/fcl/src/current-user/exec-service/strategies/tab-rpc.js

```diff
@@ -1,10 +1,9 @@
 import {uid} from "@onflow/util-uid"
 import {tab} from "./utils/tab"
 import {normalizePollingResponse} from "../../normalize/polling-response"
-import {configLens} from "../../../config-utils"
 import {VERSION} from "../../../VERSION"
 
-export function execTabRPC(service, body, opts) {
+export function execTabRPC(service, body, opts, config = {}) {
```

Comment:
Do you need config = {} if defaults are added in execService? execTabRPC(service, signable, opts, config)
should be same for all service methods I think